dashboard/src/modules/components/DatePickerComponent/index.jsx

```diff
@@ -18,17 +18,15 @@ function DatePickerWidget({
   setDateRange,
 }) {
   const [fromDate, setFromDate] = useState({});
-  const [toDate, setToDate] = useState(
-    constructUTCDate(new Date(formatDate(new Date())))
-  );
   const [strDate, setStrDate] = useState(formatDate(new Date()));
+  const [toDate, setToDate] = useState(constructUTCDate(new Date(strDate)));
   const toValidator = (date) =>
     date >= fromDate
       ? ""
       : "To date must be greater than or equal to from date";
 
   const onFromChange = (_str, date) => {
-    setFromDate(constructUTCDate(new Date(_str)));
+    setFromDate(isValidDate(date) ? constructUTCDate(new Date(_str)) : {});
```

Comment:
This is OK, but, if the entered date is not valid, do we really want to change `fromDate` at all?  (See [this comment](https://github.com/distributed-system-analysis/pbench/pull/2733#discussion_r862940868).)

Reply:
I am doing this purposely.The problem is if the user enters a ill formatted date (like '2022-03-    15'),it is also getting converted to a valid Date object ,which I don't want.Hence,I am changing the fromDate to an empty object,so that he is unable to filter using the ill formatted date and hence fix it